concepts/05 Widgets/Form/10 Organize Simple Items/05 In Groups/01 Create a Group.md

```diff
@@ -65,6 +65,96 @@ In the context of the **Form** widget, a group is called ["group item"](/api-ref
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <dx-form
+            :form-data="employee">
+            <dx-item
```

Comment:
Instead of DxItem, there are special configuration components for different form item types. Check out the [demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Common/FormsAndMultiPurposeOverview/Vue/Light/)

Reply:
Fixed